drivers/fpga/xrt/lib/subdev.c

```diff
@@ -15,9 +14,9 @@
 #include "metadata.h"
 
 #define IS_ROOT_DEV(dev) ((dev)->bus == &pci_bus_type)
```

Comment:
we need to change this code to dev->bus != &xrt_bus_type here and remove include of linux/pci.h, so that we can remove reference of pci stuff in our infra code completely. thus, we can address Tom's comments about separating infra code and pci specific implementation.